### ISSUE TITLE
[Bug]: CLI no longer works when starting a Bottle

### ISSUE THREAD

**Issue by paddylandau** [bottlesdevs/Bottles#3599]
### Describe the bug

As from yesterday's update to Bottles version 51.16, when I start a Bottle from the CLI, it no longer works.

(First discussed [on Reddit](https://redd.it/1hdcyts), where someone else reported the same problem.)

When I start Bottles itself from the menu, and then start the chosen Bottle from there, my Bottle does work, so I at least have a workaround for now.

I'm using flatpak. Here is the command that I have been using. I know that the command hasn't changed, because it's used in a script that I haven't touched in a long time:

    flatpak run --command=bottles-cli -- com.usebottles.bottles run --bottle=Quicken --executable=${HOME}/Documents/Bottles/Imported_Quicken/drive_c/quickenw/QW.EXE

It used to start the app without problem. Now, it does nothing! The only message is:

    13:36:32 (INFO) Forcing offline mode

Looking at the help documents, I see that `--command` is no longer used, so I adjusted this to:

    flatpak run -- com.usebottles.bottles run --bottle=Quicken --executable=${HOME}/Documents/Bottles/Imported_Quicken/drive_c/quickenw/QW.EXE

This is an improvement of sorts. It displays the GUI prompt to select a Bottle, which I do, but then still nothing happens. The messages are as follows:

    (bottles:2): Gtk-WARNING **: 13:37:58.633: Theme parser warning: style.css:184:3-185:1: Expected ';' at end of block
    (bottles:2): Gtk-WARNING **: 13:37:58.633: Theme parser warning: style-dark.css:23:3-24:1: Expected ';' at end of block
    13:37:58 (INFO) Launching with URI: ['run'] 
    13:37:58 (INFO) Forcing offline mode 
    MESA-INTEL: warning: ../src/intel/vulkan/anv_formats.c:763: FINISHME: support YUV colorspace with DRM format modifiers
    MESA-INTEL: warning: ../src/intel/vulkan/anv_formats.c:794: FINISHME: support more multi-planar formats with DRM modifiers

As stated above, I do at least have a workaround by starting Bottles manually and then running the executable.

I am using Linux Ubuntu 22.04.

Do you know what I can do to fix this, please, so that I can run a Bottle from the command line?

### To Reproduce

Run the CLI command to load a Bottle.

    flatpak run -- com.usebottles.bottles run --bottle=[bottle_name] --executable=[path_to_executable]

Do this from a terminal to see the error messages.

Look at running jobs using `ps -e` or equivalent. Note that none of Bottles, WINE and your chosen app is running.

### Package

Flatpak from Flathub

### Distribution

Ubuntu 22.04

### Debugging Information

```shell
Official Package: true
Version: '51.16'
DE/WM: ubuntu-xorg
Display:
    X.org: true
    X.org (port): :0
    Wayland: false
Graphics:
    vendors:
        intel:
            vendor: intel
            envs:
                DRI_PRIME: '1'
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/intel_icd.x86_64.json:/usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/intel_hasvk_icd.x86_64.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/intel_hasvk_icd.i686.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/intel_icd.i686.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 6.8.0-49-generic
Disk:
    Total: 8152485888
    Free: 8152309760
RAM:
    MemTotal: 15.2GiB
    MemAvailable: 11.5GiB
Bottles_envs: null
```

### Troubleshooting Logs

```shell
(bottles:2): Gtk-WARNING **: 13:37:58.633: Theme parser warning: style.css:184:3-185:1: Expected ';' at end of block
(bottles:2): Gtk-WARNING **: 13:37:58.633: Theme parser warning: style-dark.css:23:3-24:1: Expected ';' at end of block
13:37:58 (INFO) Launching with URI: ['run'] 
13:37:58 (INFO) Forcing offline mode 
MESA-INTEL: warning: ../src/intel/vulkan/anv_formats.c:763: FINISHME: support YUV colorspace with DRM format modifiers
MESA-INTEL: warning: ../src/intel/vulkan/anv_formats.c:794: FINISHME: support more multi-planar formats with DRM modifiers
```

### Additional context

This stopped working when Bottles updated to version 51.16

**Comment by sethchev** [comment]
I have the same issue with 51.16. Fedora 41 using flatpak. Rolled Bottles flatpak back to 51.15 and CLI bottle launch works again.

**Comment by jntesteves** [comment]
Should be fixed in 51.17 hotfix released yesterday, it's already available on Flathub. I'm closing the issue, but feel free to reopen if you still have issues after the update. Thanks for reporting.

**Comment by paddylandau** [comment]
Thank you for your work, @jntesteves 

Unfortunately, I'm still getting the same results 🙁

The error message has changed, though. Here's the new one.

### First

This is what happens when I run the command (this message is the same as before):

    (bottles:2): Gtk-WARNING **: 15:53:21.889: Theme parser warning: style.css:184:3-185:1: Expected ';' at end of block
    (bottles:2): Gtk-WARNING **: 15:53:21.889: Theme parser warning: style-dark.css:23:3-24:1: Expected ';' at end of block
    15:53:21 (INFO) Launching with URI: ['run'] 
    15:53:21 (INFO) Forcing offline mode 
    MESA-INTEL: warning: ../src/intel/vulkan/anv_formats.c:782: FINISHME: support YUV colorspace with DRM format modifiers
    MESA-INTEL: warning: ../src/intel/vulkan/anv_formats.c:814: FINISHME: support more multi-planar formats with DRM modifiers

### Second

That brings up the GUI to select which Bottle to choose. Once I select the Bottle, this is what is shown:

    15:53:25 (INFO) Forcing offline mode 
    Traceback (most recent call last):
      File "/app/bin/bottles-cli", line 758, in <module>
        cli = CLI()
              ^^^^^
      File "/app/bin/bottles-cli", line 236, in __init__
        self.__process_args()
      File "/app/bin/bottles-cli", line 285, in __process_args
        self.run_program()
      File "/app/bin/bottles-cli", line 699, in run_program
        WineExecutor.run_program(bottle, program)
                                         ^^^^^^^
    UnboundLocalError: cannot access local variable 'program' where it is not associated with a value

If I can do something to help, let me know.

### Debugging Information

    Official Package: true
    Version: '51.17'
    DE/WM: ubuntu-xorg
    Display:
        X.org: true
        X.org (port): :0
        Wayland: false
    Graphics:
        vendors:
            intel:
                vendor: intel
                envs:
                    DRI_PRIME: '1'
                icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/intel_icd.x86_64.json:/usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/intel_hasvk_icd.x86_64.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/intel_hasvk_icd.i686.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/intel_icd.i686.json
        prime:
            integrated: null
            discrete: null
    Kernel:
        Type: Linux
        Version: 6.8.0-49-generic
    Disk:
        Total: 8152489984
        Free: 8152313856
    RAM:
        MemTotal: 15.2GiB
        MemAvailable: 11.6GiB
    Bottles_envs: null

**Comment by jntesteves** [comment]
I just realized only the case of launching a `--program <name>` was fixed in #3596, but here you're trying to launch with `--executable <path>`, and that is broken. This may be the same issue as #3601. Thanks for the log, I can reproduce, and I think I know what's wrong now, I'll try to fix this asap.

**Comment by paddylandau** [comment]
Thank you for your speedy responses, @jntesteves 

I was unaware of the `--program` option. I've just tried it, and it works.

Thank you again
